docs/SQL_SERVER_WC.md

```diff
@@ -44,7 +44,7 @@ Almost all geospatial data can be converted to SQL Server format without losing
 
 #### Approximated types
 
-There is one type that Sno supports that has no SQL Server equivalent - the `interval`. This type is "approximated" as a string (or more precisely, as an `NVARCHAR`) in the SQL Server working copy while keeping its original type in the Sno dataset. Sno creates a column of type `NVARCHAR` in the appropriate place in the database table, and fills it with with the intervals formatted as [ISO8601 durations](https://en.wikipedia.org/wiki/ISO_8601#Durations). When the working copy is committed, Sno converts the contents of those columns back to `interval`, instead of their actual type, `NVARCHAR`. Since the change from `interval` to `NVARCHAR` is just a limitation of the working copy, this apparent change in type will not show up as a change in the commit log, nor will it show up as an uncommitted change if you run `sno status` to see what local changes you have made but not yet committed.
+There is one type that Sno supports that has no SQL Server equivalent - the `interval`. This type is "approximated" as `NVARCHAR` in the SQL Server working copy. See APPROXIMATED_TYPES.md for more information.
 
 #### Geometry types
```

Comment:
worth adding something to say the Geography type isn't currently supported but we're planning on it?

Reply:
Done